test/e2e/e2e_v1.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/kubernetes-sigs/kubebuilder/test/e2e/framework"
+	e2einternal "github.com/kubernetes-sigs/kubebuilder/test/internal/e2e"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("v1 main workflow", func() {
+	It("should perform main kubebuilder workflow successfully", func() {
+		testSuffix := framework.RandomSuffix()
+		c := initConfig(testSuffix)
+		kubebuilderTest := e2einternal.NewKubebuilderTest(c.workDir, framework.TestContext.BinariesDir)
+
+		prepare(c.workDir)
+		defer cleanupv1(kubebuilderTest, c.workDir, c.controllerImageName)
+
+		var controllerPodName string
+
+		output, errrr := exec.Command("which", "kubebuilder").Output()
+		fmt.Printf("output of which: %s", output)
+		Expect(errrr).NotTo(HaveOccurred())
+
+		By("init v1 project")
+		initOptions := []string{
+			"--project-version", "v1",
```

Comment:
Can you add "--project-version", "v0" to e2e_v0.go? The default init is going to change from `v0` to `v1`. You can avoid the impact by adding this flag.

Reply:
Done